disperser/dataapi/server.go

```diff
@@ -358,6 +367,38 @@ func (s *server) FetchNonSigners(c *gin.Context) {
 	c.JSON(http.StatusOK, metric)
 }
 
+// FetchDeRegisteredOperators godoc
+//
+//	@Summary	Fetch list of DeregisteredOperators Last 14 days
+//	@Tags		OperatorsInfo
+//	@Produce	json
+//	@Success	200		{object}	BlobsResponse
+//	@Failure	400		{object}	ErrorResponse	"error: Bad request"
+//	@Failure	404		{object}	ErrorResponse	"error: Not found"
+//	@Failure	500		{object}	ErrorResponse	"error: Server error"
+//	@Router		/operatorsInfo/deRegisteredLast14days [get]
+func (s *server) FetchDeRegisteredOperators(c *gin.Context) {
```

Comment:
We may not be interested in all deregistered operators, not even those registered in past 14 days (although it's a starting point for now). What really matters is operators in "lame duck": the last batch they signed is in past 14 days.